vms/platformvm/txs/fee/calculator_test.go

```diff
@@ -3,251 +3,127 @@
-	"github.com/ava-labs/avalanchego/vms/platformvm/upgrade"
-)
-
-func TestTxFees(t *testing.T) {
-	feeTestsDefaultCfg := StaticConfig{
+var (
+	testStaticConfig = StaticConfig{
 		TxFee:                         1 * units.Avax,
-		CreateAssetTxFee:              2 * units.Avax,
-		CreateSubnetTxFee:             3 * units.Avax,
-		TransformSubnetTxFee:          4 * units.Avax,
-		CreateBlockchainTxFee:         5 * units.Avax,
-		AddPrimaryNetworkValidatorFee: 6 * units.Avax,
-		AddPrimaryNetworkDelegatorFee: 7 * units.Avax,
-		AddSubnetValidatorFee:         8 * units.Avax,
-		AddSubnetDelegatorFee:         9 * units.Avax,
+		CreateSubnetTxFee:             2 * units.Avax,
+		TransformSubnetTxFee:          3 * units.Avax,
+		CreateBlockchainTxFee:         4 * units.Avax,
+		AddPrimaryNetworkValidatorFee: 5 * units.Avax,
+		AddPrimaryNetworkDelegatorFee: 6 * units.Avax,
+		AddSubnetValidatorFee:         7 * units.Avax,
+		AddSubnetDelegatorFee:         8 * units.Avax,
 	}
 
-	latestForkTime := time.Unix(1713945427, 0)
-	upgrades := upgrade.Config{
-		EUpgradeTime:      latestForkTime,
-		DurangoTime:       latestForkTime.Add(-1 * time.Hour),
-		CortinaTime:       latestForkTime.Add(-2 * time.Hour),
-		BanffTime:         latestForkTime.Add(-3 * time.Hour),
-		ApricotPhase5Time: latestForkTime.Add(-4 * time.Hour),
-		ApricotPhase3Time: latestForkTime.Add(-5 * time.Hour),
-	}
-
-	// chain times needed to have specific upgrades active
-	preEUpgradeTime := upgrades.EUpgradeTime.Add(-1 * time.Second)
-	preApricotPhase3Time := upgrades.ApricotPhase3Time.Add(-1 * time.Second)
-
-	tests := []struct {
-		name       string
-		chainTime  time.Time
-		unsignedTx func() txs.UnsignedTx
-		expected   uint64
+	txTests = []struct {
+		name                 string
+		tx                   string
+		expectedStaticFee    uint64
+		expectedStaticFeeErr error
 	}{
 		{
-			name:       "AddValidatorTx pre EUpgrade",
-			chainTime:  preEUpgradeTime,
-			unsignedTx: addValidatorTx,
-			expected:   feeTestsDefaultCfg.AddPrimaryNetworkValidatorFee,
-		},
-		{
-			name:       "AddSubnetValidatorTx pre EUpgrade",
-			chainTime:  preEUpgradeTime,
-			unsignedTx: addSubnetValidatorTx,
-			expected:   feeTestsDefaultCfg.AddSubnetValidatorFee,
-		},
-		{
-			name:       "AddDelegatorTx pre EUpgrade",
-			chainTime:  preEUpgradeTime,
-			unsignedTx: addDelegatorTx,
-			expected:   feeTestsDefaultCfg.AddPrimaryNetworkDelegatorFee,
+			name:                 "AdvanceTimeTx",
+			tx:                   "0000000000130000000066a56fe700000000",
+			expectedStaticFee:    0,
+			expectedStaticFeeErr: ErrUnsupportedTx,
 		},
 		{
-			name:       "CreateChainTx pre ApricotPhase3",
-			chainTime:  preApricotPhase3Time,
-			unsignedTx: createChainTx,
-			expected:   feeTestsDefaultCfg.CreateAssetTxFee,
+			name:                 "RewardValidatorTx",
+			tx:                   "0000000000143d0ad12b8ee8928edf248ca91ca55600fb383f07c32bff1d6dec472b25cf59a700000000",
```

Comment:
Do you think it would be better to move these transactions to separate files (and embed them?). These are really more intended to be compliance tests that should never change... I feel like if we included them in their non-binary forms... Their definition would dominate the tests... When really I think of them as amorphous tests to prevent unexpected changes.